home/containers/Project.tsx

```diff
@@ -31,86 +31,34 @@ export type ProjectDataProject = {
   slug: string;
   fullName: string;
   username: string;
-  published: string;
+  published: boolean;
   description: string;
-  githubUrl?: string;
-  playStoreUrl?: string;
-  appStoreUrl?: string;
+  githubUrl?: string | null;
+  playStoreUrl?: string | null;
+  appStoreUrl?: string | null;
```

Comment:
Should these be non-optional fields now that they are nullable?

Reply:
In the generated typedef they are both optional and `Maybe<>`, which corresponds to nullable. So for this to be compatible with the typedef we need to make these both optional (undefined) and nullable.